packages/integration/package.json

```diff
@@ -20,7 +20,8 @@
     "find-up": "^5.0.0",
     "javascript-stringify": "^2.0.1",
     "lodash": "^4.17.21",
-    "outdent": "^0.8.0"
+    "outdent": "^0.8.0",
+    "ts-deepmerge": "^2.0.1"
```

Comment:
We already have a `deepmerge` dep in the core `@vanilla-extract/css` package. Could we just use the same one here?

Reply:
I ended up getting rid of the `deepmerge` altogether now there is a known set of options.